backend/src/main/java/com/example/backend/appuser/AppUserController.java

```diff
@@ -8,7 +8,7 @@
 import java.util.Optional;
 
 @RestController
-@RequestMapping("/api/appuser")
+@RequestMapping("/api/app-user")
```

Comment:
Resource REST endpoints should always be written in plural. `/api/app-users`

Reply:
_Changed_